incubator-tools/test_harness_guide/test_harness_guide.ipynb

```diff
@@ -0,0 +1,1455 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #490.    def batch_process_documents(
</pre><p>Not sure why this extra functuion is needed</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/700/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>In updated notebook, this staging function removed.</p>